README.md

```diff
@@ -75,7 +75,7 @@ In the base case, we run the T1w scan through the model for prediction.
 ```bash
 # Get sample T1w scan.
 wget -nc https://dl.dropbox.com/s/g1vn5p3grifro4d/T1w.nii.gz
-docker run --rm -v $PWD:/data kaczmarj/nobrainer \
+docker run --rm -v $PWD:/data -v nobrainer-models:/models kaczmarj/nobrainer \
```

Comment:
shouldn't there be some instruction as to how to get nobrainer-models?

Reply:
It is mentioned in the section 'Using pre-trained networks' about how 'Pre-trained networks are available in the Nobrainer models repository'.
Do we need a 'git clone' instruction also then to obtain no-brainer models?

Reply:
i don't think that is sufficient for a person to follow. how would a new person carry out the steps?

Reply:
@wazeerzulfikar - could you please update this PR to address my comment on how a new person would go about using this?

Reply:
Yes, I am working on it.